ui-pf4/src/main/webapp/src/components/select-card-gallery/select-card-gallery.tsx

```diff
@@ -93,6 +96,12 @@ export const SelectCardGallery: React.FC<SelectCardGalleryProps> = ({
   value,
   onChange,
 }) => {
+  // When Option is an array e.g. EAP Card then it has a default value
+  // However the default value changes after the user selects the dropdown.
+  // 'tempDefault' will keep the last selected dropdown so doesn't go back to
+  // the original 'default' value. KEY=card.label, value='last card's dropdown selected value'
+  const [tempDefault, setTmpDefault] = useState<Map<string, string>>(new Map());
```

Comment:
I think that `tempDefault` might be a misleading name.
In the end, a "temporary default" is the "selected" value I think (also from comment, it's the "last selected")
So, could you rename `tempDefault` with `selected`?
and `setTmpDefault` with `setSelected`?